README.md

```diff
@@ -1,90 +1,141 @@
-
-In particular, we would recommend that you read the articles in the following order:
-
-* [Get Started](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/ospsuite.html)
-* [Loading a simulation and accessing entities](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/load-get.html)
-* [Changing parameter and molecule start values](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/set-values.html)
-* [Running a simulation](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/run-simulation.html)
-* [Efficient calculations](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/efficient-calculations.html)
-* [Creating individuals](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-individual.html)
-* [Population simulations](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-run-population.html)
-* [PK Analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/pk-analysis.html)
-* [Sensitivity analysis](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/sensitivity-analysis.html)
-* [Table parameters](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/table-parameters.html)
-* [Dimensions and Units](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/unit-conversion.html)
-* [Working with data sets and import from excel](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/observed-data.html)
-* [Working with `DataCombined` class](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined.html)
-* [Visualizations with `DataCombined`](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/data-combined-plotting.html)
-
-# Installation
-
-The **ospsuite-R** package is compatible with version 3.6.x **AND** version 4.x.x of R. One of its dependency, **rClr** needs to be installed specifically for the targeted R version. Please follow the installation instructions below.
-
-**ospsuite** requires following packages to be installed:
-
-From CRAN:
-
-- [dplyr](https://cran.r-project.org/web/packages/dplyr/index.html)
-- [purrr](https://cran.r-project.org/web/packages/purrr/index.html)
-- [R6](https://cran.r-project.org/web/packages/R6/index.html)
-- [readr](https://cran.r-project.org/web/packages/readr/index.html)
-- [stringr](https://cran.r-project.org/web/packages/stringr/index.html)
-- [tidyr](https://cran.r-project.org/web/packages/tidyr/index.html)
-- [ggplot2](https://cran.r-project.org/web/packages/ggplot2/index.html)
-- [rlang](https://cran.r-project.org/web/packages/rlang/index.html)
-- [jsonlite](https://cran.r-project.org/web/packages/jsonlite/index.html)
-- [patchwork](https://cran.r-project.org/web/packages/patchwork/index.html)
-- [cowplot](https://cran.r-project.org/web/packages/cowplot/index.html)
-- [scales](https://cran.r-project.org/web/packages/scales/index.html)
-- [data.table](https://cran.r-project.org/web/packages/data.table/index.html)
```

Comment:
why removing CRAN packages from the installation instructions? 
Are they installed automatically somehow? AFAIK not

Reply:
We need some automated way of listing the dependencies, working on it now. It will be very hard to maintain the list manually.